### PR TITLE
Feature/new deps

### PR DESCRIPTION
**Pull request by codepfleger**

### CODE REVIEW

File: trasier-client-api-protobuf/pom.xml [hunk 1]
```diff
@@ -3,25 +3,25 @@
...
-        <version>2.0.2-SNAPSHOT</version>
+        <version>2.1.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>trasier-client-api-protobuf</artifactId>
 
     <properties>
-        <grpc.version>1.16.1</grpc.version>
+        <grpc.version>1.24.0</grpc.version>
```
Comment (jlupi) [inline trasier-client-api-protobuf/pom.xml]: in parent it is still 1.16.1, can we remove it from over there?

File: trasier-client-core/src/main/java/com/trasier/client/http/TrasierHttpClientHandler.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            failure.incrementAndGet();
+        }
+        return super.onStatusReceived(status);
+    }
+
+    @Override
+    public Response onCompleted(Response response) throws Exception {
+        long currentTime = System.currentTimeMillis();
+        if (nextLog < currentTime) {
+            synchronized (success) {
```
Comment (jlupi) [inline trasier-client-core/src/main/java/com/trasier/client/http/TrasierHttpClientHandler.java]: do we have to synchronize on atomics? an lock object would be better no?

File: trasier-client-spring-grpc/pom.xml [hunk 3]
```diff
@@ -34,11 +34,12 @@
             <groupId>org.iq80.snappy</groupId>
             <artifactId>snappy</artifactId>
         </dependency>
-        <dependency>
-            <groupId>com.google.auth</groupId>
-            <artifactId>google-auth-library-oauth2-http</artifactId>
-            <version>${google-auth-library-oauth2-http.version}</version>
-        </dependency>
+
+<!--        <dependency>-->
```
Comment (jlupi) [inline trasier-client-spring-grpc/pom.xml]: commented out code

File: trasier-client-api-protobuf/pom.xml (hunk 1 above)
Comment (codepfleger) [reply trasier-client-api-protobuf/pom.xml]: done

File: trasier-client-core/src/main/java/com/trasier/client/http/TrasierHttpClientHandler.java (hunk 2 above)
Comment (codepfleger) [reply trasier-client-core/src/main/java/com/trasier/client/http/TrasierHttpClientHandler.java]: done

File: trasier-client-spring-grpc/pom.xml (hunk 3 above)
Comment (codepfleger) [reply trasier-client-spring-grpc/pom.xml]: done

**Pull request merged**